### ISSUE TITLE
Issue with SA1003 cast and ':' operator must not appear at the end of a line.

### ISSUE THREAD

**Issue by mpawelski** [DotNetAnalyzers/StyleCopAnalyzers#2049]
Such code:
```
var loggerFactory =
                (ILoggerFactory)
                GlobalConfiguration.Configuration.DependencyResolver.GetService(typeof(ILoggerFactory));
```
gives error:
`Error	SA1003	Operator '(ILoggerFactory)' must not appear at the end of a line.`

and this code:
```
class SomeClass : SomeBaseClass
{
    public NullExpiryYearMonth() : 
        base(
        null, 
        null)
    {
    }
}
```
gives error:
`Error	SA1003	Operator ':' must not appear at the end of a line.	`

In both cases it didn't give error in StyleCop Classic. I think we should change it to comply with StyleCop classic.

**Comment by sharwell** [comment]
This is an duplicate of #1300 (which I was surprised to see, considering you reported two separate items).

**Comment by mpawelski** [comment]
Sorry about that!
I was so happy to finish migration of existing project to StyleCop.Analyzers that I quickly wanted to give feedback about this rule I had issues with. I forgot to check if it was already mentioned before. Stupid me :(
I agree with with what was discussed in #1300.

BTW thanks for working on this project! I was able to migrate exisiting solution with old StyleCop and now I can use VS2015 with new c#6 features and Stylecop rules. Great work!

**Comment by sharwell** [comment]
@mpawelski Duplicates are not a problem, I just didn't want to see your feedback get lost. :smile:
